packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -839,39 +872,44 @@ test('authorized group is allowed to listen to onUpdate', async () => {
         }
       }
     `,
-    fetchPolicy: 'network-only',
-  });
+    authMode: GRAPHQL_AUTH_MODE.AMAZON_COGNITO_USER_POOLS,
+  }) as Observable<any>;
 
   const subscriptionPromise = new Promise((resolve, reject) => {
     const subscription = observer.subscribe({
       next: event => {
-        subscription.unsubscribe();
-        const subResponse = event.data.onUpdateMember;
+        const subResponse = event.value.data.onUpdateMember;
         subscription.unsubscribe();
         expect(subResponse).toBeDefined();
         expect(subResponse.id).toEqual(memberID);
-        expect(subResponse.name).toEqual(memberName);
+        expect(subResponse.name).toEqual(newMemberName);
         resolve(undefined);
       },
       complete: () => {},
       error: err => {
-        console.log(err);
-        reject(err);
+        console.log(JSON.stringify(err, null, 4));
+        resolve(err);
```

Comment:
Why is the Promise resolving here instead of rejecting?

Reply:
That's a typo